front50-web/src/main/java/com/netflix/spinnaker/front50/controllers/PipelineController.java

```diff
@@ -249,13 +249,12 @@ public synchronized Pipeline save(
     pipeline.setName(pipeline.getName().trim());
     pipeline = ensureCronTriggersHaveIdentifier(pipeline);
 
-    if (Strings.isNullOrEmpty(pipeline.getId())) {
+    if (Strings.isNullOrEmpty(pipeline.getId())
+        || (boolean) pipeline.getAny().getOrDefault("regenerateCronTriggerIds", false)) {
       // ensure that cron triggers are assigned a unique identifier for new pipelines
-      List<Trigger> triggers = pipeline.getTriggers();
-      triggers.stream()
+      pipeline.getTriggers().stream()
```

Comment:
https://github.com/spinnaker/front50/pull/987/files#r515405296 is no longer valid after https://github.com/spinnaker/front50/pull/1035/files#diff-0b2bc300fd3965c64ba4184955384aa8609cace04e1c015b31ef4a83552e53d1R36.  There's no need to get/modify/set anymore.